src/routes/singleProject/[id]/+page.svelte

```diff
@@ -435,24 +435,8 @@
             <p>No resources</p>
           {/if}
 
-          <!-- <Resources
-          username="@username392"
-          title="UX Case Study for an Internship App in Sierra Leone"
-          description="A brief summary of what the resource is about."
-          likes={187}
-          comments={64}
-        >
-          <span slot="header">My Resources</span>
-          <img slot="icon" src="icon.png" alt="Resource icon" />
-          <img slot="profile-icon" src="profile-icon.png" alt="User" />
-        </Resources> -->
```

Comment:
lets favour deleting rather than commenting out.

with a clean commit, we can always get the code back